cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -281,6 +282,24 @@ func (c *CStorVolumeReplicaController) cVREventHandler(
 		if isCVRCreateStatus(cvrObj) {
 			return c.cVRAddEventHandler(cvrObj, fullVolName)
 		}
+
+		if cvrObj.Annotations[restoreCompletedAnnotation] == "true" {
+			targetIp, err := volumereplica.GetTargetIp(fullVolName)
+			if err != nil {
+				return "", err
```

Comment:
We need to log error here.
If returned status is empty then caller is ignoring the error. https://github.com/openebs/maya/blob/master/cmd/cstor-pool-mgmt/controller/replica-controller/handler.go#L134

Reply:
Also, logging this as an event on CVR will ease the debugging.

Reply:
done, PTAL